benchopt/base.py

```diff
@@ -103,59 +102,42 @@ def is_installed(cls, env_name=None):
 
     @classmethod
     def install(cls, env_name=None, force=False):
-        """Install the class in the given virtual env.
+        """Install the class in the given conda env.
 
         Parameters
         ----------
         env_name: str or None
-            Name of the environment where the class should be installed. If
+            Name of the conda env where the class should be installed. If
             None, tries to install it in the current environment.
         force : boolean (default: False)
-            If set to True, first tries to uninstall the class from the
-            environment before installing it.
+            If set to True, forces reinstallation when using conda.
 
         Returns
         -------
         is_installed: bool
             True if the class is correctly installed in the environment.
         """
-        # uninstall the class that requires a force reinstall
-        if force:
-            cls.uninstall(env_name=env_name)
+        print(f"Installing {cls.name} in {env_name}:...",
+              end='', flush=True)
+        try:
```

Comment:
You remove the skip when install. This would avoid the long dependency resolution + update when the package is already installed so I would rather continue to skip if it is already installed no?

Reply:
I did so because conda or pip will skip install if requirement is already satisfied. Do you think it is more explicit to keep the skip ? It does not cost much indeed.